client/src/App.js

```diff
@@ -1,13 +1,28 @@
 import React from "react";
+import { BrowserRouter, Redirect, Route, Switch } from "react-router-dom";
 
+import LandingPage from "./components/LandingPage";
 import Header from "./components/Header";
 import "./App.css";
 
 function App() {
     return (
-        <div className="App">
-            <Header />
-        </div>
+        <BrowserRouter>
+            <Switch>
+                <Route exact path="/">
+                    <Redirect to="/signup" />
+                </Route>
+
+                <Route exact path="/signup">
+                    <LandingPage />
+                </Route>
+
+                <Route exact path="/home">
+                    {/* Replace following component with actual Home component when created */}
+                    <Header />
+                </Route>
+            </Switch>
+        </BrowserRouter>
     );
```

Comment:
Can do check for whether or not the user is signed in or not later down the track but this is good for the mock